.gitignore

```diff
@@ -348,3 +361,28 @@ MigrationBackup/
 
 # Ionide (cross platform F# VS Code tools) working folder
 .ionide/
+
+# Fody - auto-generated XML schema
+FodyWeavers.xsd
+
+# VS Code files for those working on multiple tools
+.vscode/*
+!.vscode/settings.json
+!.vscode/tasks.json
+!.vscode/launch.json
+!.vscode/extensions.json
+*.code-workspace
```

Comment:
I would just keep these ones and limit to only the used language/development scope in this repo which is atm only .bicep/.json/.yaml/.yml and .ps1. 
What would be or are the use cases to ignore the other ones?